src/Account/index.jsx

```diff
@@ -0,0 +1,50 @@
+// Account.jsx
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Card, CardHeader, Text, Spinner } from '@fluentui/react-components';
+import fetchAccount from '../services/account';
+import './styles.css';
```

Comment:
Let's make this a styles.module.css import and use the class object approach

```
import styles from 'styles.module.css'
...
<div className={styles.container}
```

Otherwise this css impact all scopes